lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -805,6 +805,7 @@ public Object handleResponse(HttpCore.Response response, ErrorInfo error) throws
                 /* append all relevant params to token params */
                 Map<String, Param> urlParams = null;
                 URL authUrl = HttpUtils.parseUrl(authOptions.authUrl);
+                final String querylessUrl = HttpUtils.removeQueryFromURL(authOptions.authUrl);
```

Comment:
can be renamed to `final String cleanUrlWithoutQueryParams`

Reply:
Thanks. I made names a bit more clear : dd09c50d5e7ecbb1e8a44aeea03947df84623763